MIT.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [2022] [NatNael]
```

Comment:
- [ ] Good job adding the MIT file, The problem is you need to update its content in regard to your detail and the right year. Please replace the [YEAR] with the current year.